test.js

```diff
@@ -24,6 +24,9 @@ assert.equal(ip.range("1.2.3").errors, "IP must contain 4 octets, contained: 3")
 assert.equal(ip.range("1.2.3.a").errors, "error in octet 4, invalid octet spec: 'a'");
 assert.equal(ip.range("1.2.3.266").errors, "error in octet 4, octet range out of bounds: '266'");
 
+// check valid subnet
+assert.equal(ip.range("52.123.5-3.*").errors, "IP subnet mask error, check start and end range: 52.123.5.0 - 52.123.3.255");
+assert.equal(ip.range("52.123.3.5-2").errors, "IP subnet mask error, check start and end range: 52.123.3.5 - 52.123.3.2");
```

Comment:
"masks" was originally a poorly named variable for a CIDR notation IP range.  It was a mistake for me to name the variable that.  Let's not compound the problem with an error message that talks about masks when the input isn't a mask at all.  Also, I think we should repeat the input that generated the error, rather than report on an an internal interpretation of the input.  

It looks like you're catching the case where an octet range is specified with the `-` range in descending order.  If so, let's just check for that specifically and emit an error message saying that is wrong.

If you're just trying to add a catch all for when the ip calculator module emits an error, let's just stick to something generic like "internal error in ip-subnet-calculator for ip range value: %s" and show the input that produced the error, rather than an intermediate interpretation like fromAddress and toAddress.